frontend/fingertips-frontend/components/pages/search/results/index.tsx

```diff
@@ -11,6 +28,16 @@ export function SearchResults({ indicator }: Readonly<{ indicator: string }>) {
       />
       <H1>Search results</H1>
       <Paragraph>{`You searched for indicator "**${indicator}**"`}</Paragraph>
+      {searchResults.length ? (
+        <UnorderedList listStyleType="none">
+          {searchResults.map((result) => (
+            <SearchResult key={result.id} result={result} />
+          ))}
+          <SectionBreak visible="true" />
```

Comment:
Looks like you have a `SectionBreak` in the `SearchResult` component itself too. Why do we need both? It seems odd to have one before in that component and one after the component here.

Reply:
I can see you've not got one `SectionBreak` at the start of the list and then each result adds its own after its content, so I'm happy to resolve this.